### DISCUSSION TITLE
docker image

### DISCUSSION THREAD

**Discussion by punkyard** [chamilo/chamilo-lms#5218] [General]
Hi
is docker image a subject at chamilo?

**Comment by ywarnier** [comment]
The https://github.com/chamilo/docker-chamilo project is there for that, but the image is a bit old now. You would have to edit the Dockerfile with version 1.11.26 and fix the PHP version to 7.4, but I'm not completely sure if that would work (I think so, not a lot of other things have changed in the install requirements since then).
